pkgs/by-name/ln/lnav/package.nix

```diff
@@ -52,6 +58,15 @@ stdenv.mkDerivation rec {
     libarchive
   ];
 
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = "${src}/src/third-party/prqlc-c/Cargo.lock";
```

Comment:
I'm pretty sure this is IFD. Use the new `rustPlatform.fetchCargoVendor` instead if there are git deps.